test/JDBC/upgrade/latest/schedule

```diff
@@ -296,6 +296,8 @@ sys-index_columns
 sys-indexes
 sys-key_constraints
 sys-master_files
+sys-partitions
```

Comment:
this will only test changes in Major Version Upgrade, to validate for minor upgrade add to 14.6 schedule file also